src/Common/tests/System/PlatformDetection.cs

```diff
@@ -385,5 +385,11 @@ public static bool IsNonZeroLowerBoundArraySupported
         }
 
         private static volatile Tuple<bool> s_lazyNonZeroLowerBoundArraySupported;
+
+        public static bool IsReflectionEmitSupported = !PlatformDetection.IsNetNative;
+
+        // System.Security.Cryptography.Xml.XmlDsigXsltTransform.GetOutput() relies on XslCompiledTransform which relies
+        // heavily on Reflection.Emit
+        public static bool IsXmlDsigXsltTransformSupported => PlatformDetection.IsReflectionEmitSupported;
```

Comment:
This feels like it shouldn't be in the big common test detection, but rather scoped to the Crypto.Xml test library